src/App.scss

```diff
@@ -1,3 +1,11 @@
 iframe {
   display: none;
 }
+
+html {
+  background-color: white;
+}
```

Comment:
It's white by default, do you really need it?

Reply:
For devices that have a black system theme, it is not white by default.